### PR TITLE
WRP-5618: Added CPU throttling to decrease the performance of the bee server

### PR DESCRIPTION
**Pull request by alexandrumorariu**
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
I added a command that can increase the CPU throttling but it must be run from command line. Since CPU throttling was requested only for certain metrics(DCL, FCP, LCP), emulating it was added in each relevant test, instead of the file puppeteer.setup.js

ex: "npm run test -- --throttling=5".

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
This change made some of the tests fail on my local enviroment, but they pass on jenkins.

### Links
[//]: # (Related issues, references)
WRP-5618

### Comments

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,6 +29,28 @@ npm run serve
...
+```
+
+```
+npm run test-tv-throttling
+```
+
+or for a custom value use the commands:
+
+```
+npm run test --throttling=4
```
Comment (seunghoh) [inline README.md:47]: Isn't it `npm run test -- --throttling=4`?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,28 @@ npm run serve
...
+```
+
+```
+npm run test-tv --throttling=3
```
Comment (seunghoh) [inline README.md:51]: Isn't it `npm run test -- --throttling=3`?

File: package.json [hunk 3]
```diff
@@ -13,7 +13,9 @@
 		"lint": "enact lint .",
 		"license": "enact license",
 		"test": "jest --detectOpenHandles --target=PC",
+		"test-throttling": "jest --detectOpenHandles --target=PC --throttling=2",
```
Comment (seunghoh) [inline package.json:16]: Let's remove this. just --throttling option is enough

File: package.json [hunk 4]
```diff
@@ -13,7 +13,9 @@
...
 		"test-tv": "jest --detectOpenHandles --target=TV",
+		"test-tv-throttling": "jest --detectOpenHandles --target=TV --throttling=2",
```
Comment (seunghoh) [inline package.json:18]: Let's remove this. just --throttling option is enough

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling
+
+If the test environment is too good, and the metrics are too big to catch dips in performance after a PR you can test it with `CPU throttling` of 2 with the commands:
```
Comment (0x64) [inline README.md:34]: @stanca-pop-lgp @adrian-cocoara-lgp 
Could you please review again for commits after your review? This explanation looks not for README but PR description. At least we need to provide some available options and what it affects. A kind of `Throttling option` section would be better to explain what users are able to do something for their purpose.

Comment (adrian-cocoara-lgp) [reply README.md:34]: Sure, we will find a better description for this

File: README.md [hunk 6]
```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling
```
Comment (seunghoh) [inline README.md:32]: ## Throttling

File: README.md (hunk 5 above)
Comment (seunghoh) [inline README.md:34]: You can simulate a low-end device with a throttling option. Available values are... (please explain the options and also the throttling means CPU throttling etc.) 

CC @daniel-stoian-lgp

File: README.md (hunk 1 above)
Comment (alexandrumorariu) [reply README.md]: I changed the command to use the structure you suggested

File: README.md (hunk 6 above)
Comment (adrian-cocoara-lgp) [reply README.md]: In out opinion `CPU Throttling` seem a more appropiate title.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**